### ISSUE TITLE
How to calculate the allele frequency spectrum of a mixed ploidy population from a VCF file?

### ISSUE THREAD

**Issue by Song-10-YF** [isaacovercast/easySFS#70]
If I am calculating the allele frequency spectrum for a mixed ploidy group from a called VCF file, how should I set the parameters?

**Comment by isaacovercast** [comment]
Mixed ploidy like haploid and diploid? I'm not sure how the best way to do this is as I've never encountered this before. In practice if you set ploidy=1 or ploidy=2 with mixed haploid/diploid samples the results will be the same, but I would have to think more about how meaningful it would be. For polyploid samples set ploidy to the max ploidy value among all samples.

**Comment by Song-10-YF** [comment]
I have diploid and tetraploid in my vcf, so I should set it to 4?
------------------ Original ------------------
From: Isaac Overcast ***@***.***&gt;
Date: Fri,Dec 27,2024 11:46 PM
To: isaacovercast/easySFS ***@***.***&gt;
Cc: Yan-Feng Song ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [isaacovercast/easySFS] How to calculate the allele frequencyspectrum of a mixed ploidy population from a VCF file? (Issue #70)





 
Mixed ploidy like haploid and diploid? I'm not sure how the best way to do this is as I've never encountered this before. In practice if you set ploidy=1 or ploidy=2 with mixed haploid/diploid samples the results will be the same, but I would have to think more about how meaningful it would be. For polyploid samples set ploidy to the max ploidy value among all samples.

**Comment by isaacovercast** [comment]
Yes. Let me know how it goes, I have never tried this so I am curious.

**Comment by Song-10-YF** [comment]
I'll try it. But I'm not sure it's right either.


------------------ Original ------------------
From: Isaac Overcast ***@***.***&gt;
Date: Fri,Dec 27,2024 11:53 PM
To: isaacovercast/easySFS ***@***.***&gt;
Cc: Yan-Feng Song ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [isaacovercast/easySFS] How to calculate the allele frequencyspectrum of a mixed ploidy population from a VCF file? (Issue #70)





 
Yes. Let me know how it goes, I have never tried this so I am curious.
